### PR TITLE
Confirm if the requested output-format is in appsetting.json

### PR DESCRIPTION
**Pull request by masv-scb**

### CODE REVIEW

File: PxWeb/Controllers/Api2/TableApiController.cs [hunk 1]
```diff
@@ -177,11 +177,15 @@ private IActionResult GetData(string id, string? lang, VariablesSelection? varia
             }
             
             builder.BuildForPresentation(selection);
-            
+
             if (outputFormat == null)
             {
                 outputFormat = _configOptions.DefaultOutputFormat;
             }
+            else if (!_configOptions.OutputFormats.Contains(outputFormat))
```
Comment (likp) [inline PxWeb/Controllers/Api2/TableApiController.cs:185]: This compasison is case sansitive. It should be insensitive.

**Review by likp — COMMENTED** [review]
The comparison shoudl be case insenstive

**Review by MikaelNordberg — COMMENTED** [review]
If you fix Petros suggestion it is ok

File: PxWeb/Controllers/Api2/TableApiController.cs (hunk 1 above)
Comment (masv-scb) [reply PxWeb/Controllers/Api2/TableApiController.cs]: Added StringComparer.OrdinalIgnoreCase

**Comment by masv-scb** [comment]
Check if the requested output-format is in appsetting.json

**Pull request merged**